src/NotificationEventAbstract.php

```diff
@@ -141,6 +141,37 @@ public static function raise(
                                     $send_data['mode']                      = $data['mode'];
                                     $send_data['event']                     = $event;
 
+                                    $send_data['documents_data'] = '';
+                                    foreach ($options as $k => $v) {
```

Comment:
Iterating on `$options` will probably not work in many cases. Unless I miss something, the `Ticket::new` notification will not anymore have atach documents, as there is no `$options` value passed to the `NotificationEvent::raiseEvent()` method.

Maybe the `NotificationEvent::raiseEvent()` method should have a new `CommonDBTM $triggerer_item = null` parameter that would be use to determine on which item an the operation that triggered the event was made. If this param is `null`, we could fallback on the `$item` object.